MotionMark/tests/svg/suits.js

```diff
@@ -23,79 +23,82 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-var SuperSuitsParticle = window.SuitsParticle;
-ClipSuit = Utilities.createSubclass(SuperSuitsParticle,
-    function(stage)
+class ClipSuit extends SuitsParticle {
+    static get hasGradient() { return false; }
+
+    constructor(stage)
     {
+        this.isClipPath = true;
         this.initialize(stage);
-    }, {
+    }
 
-    isClipPath: true,
-    hasGradient: false,
-    move: function()
+    move()
     {
         this.element.setAttribute("transform", "translate(" + (this.position.x - this.size.center.x) + "," + (this.position.y - this.size.center.x) + ")");
     }
-});
+}
+
+class ShapeSuit extends SuitsParticle {
+    static get hasGradient() { return false; }
```

Comment:
What is `hasGradient`? And what is it used for? Can we pass it to the super?

Reply:
This is overriding the value of the static in the base class.